### PR TITLE
Rename pymc3 -> pymc 

### PR DESCRIPTION
**Pull request by fonnesbeck**
In advance of the PyMC4 beta, this PR is renaming `pymc3` to `pymc` throughout, taking care to preserve cases like `arviz.from_pymc3` and `bcm_pymc3` which have not yet changed.

### CODE REVIEW

**Review by twiecki — APPROVED** [review]

File: Rethinking/Chp_04.ipynb [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "import matplotlib.pyplot as plt\n",
     "import numpy as np\n",
     "import pandas as pd\n",
-    "import pymc3 as pm\n",
+    "import pymc as pm\n",
```
Comment (savagej) [inline Rethinking/Chp_04.ipynb:13]: With this change, I get 
```
AttributeError: module 'pymc' has no attribute 'trace_to_dataframe'
```
in code 4.32

File: Rethinking_2/Chp_04.ipynb [hunk 2]
```diff
@@ -10,7 +10,7 @@
     "import matplotlib.pyplot as plt\n",
     "import numpy as np\n",
     "import pandas as pd\n",
-    "import pymc3 as pm\n",
+    "import pymc as pm\n",
```
Comment (savagej) [inline Rethinking_2/Chp_04.ipynb:13]: With this change, I get
```
AttributeError: module 'pymc' has no attribute 'trace_to_dataframe'
```
in code 4.32

Comment (savagej) [reply Rethinking_2/Chp_04.ipynb:13]: I tried moving over to 
```
trace_df = az.from_pymc3(trace=trace_4_1)
```
but got 
```
ModuleNotFoundError: No module named 'pymc3'
```

Comment (savagej) [reply Rethinking_2/Chp_04.ipynb:13]: Oh I see pr #141 is working on this

File: Rethinking_2/Chp_02.ipynb [hunk 3]
```diff
@@ -9,7 +9,7 @@
     "import arviz as az\n",
     "import matplotlib.pyplot as plt\n",
     "import numpy as np\n",
-    "import pymc3 as pm\n",
+    "import pymc as pm\n",
```
Comment (savagej) [inline Rethinking_2/Chp_02.ipynb:12]: Issue with this chapter using pymc too: https://github.com/pymc-devs/pymc/issues/5443

**Comment by canyon289** [comment]
Should this one be merged sooner rather than later?

**Comment by twiecki** [comment]
Good point, need to resolve conflicts @fonnesbeck

**Pull request merged**